_data/resources/downloads.yml

```diff
@@ -10,3 +10,10 @@
   version: 1.0
   date: 2019-12-02
   url: UCO and CASE Shared Development Practices v1.0.pdf
+- title: Style Guide for Documentation of the CASE Ontology
+  version: 1.0
+  date: 2020-04-07
+  url: Style Guide for Documentation of the CASE Ontology.pdf
+
```

Comment:
Oh, sorry, extra blank lines snuck in from my editor.  These didn't affect my local website test.

Reply:
Blank lines have been removed in the follow-on commit.